Tests/OpenCombineTests/PublisherTests/DropWhileTests.swift

```diff
@@ -225,16 +225,12 @@ final class DropWhileTests: XCTestCase {
         publisher.send(completion: .finished)
         XCTAssertEqual(subscription.history, [.requested(.unlimited)])
         XCTAssertEqual(tracking.history, [.subscription("DropWhile"),
-                                          .completion(.finished),
                                           .completion(.finished)])
 
         publisher.send(completion: .failure(.oops))
         publisher.send(completion: .failure(.oops))
         XCTAssertEqual(tracking.history, [.subscription("DropWhile"),
-                                          .completion(.finished),
-                                          .completion(.finished),
-                                          .completion(.failure(.oops)),
-                                          .completion(.failure(.oops))])
+                                          .completion(.finished)])
```

Comment:
I like that they've fixed it, makes more sense now.

Reply:
Agreed. In one of my branches, I have a bunch of tests confirming how Apple behaves in all of these "received x after completion" scenarios.